src/portablexdr.mk

```diff
@@ -20,9 +20,8 @@ endef
 define $(PKG)_BUILD
     cd '$(1)' && ./configure \
         $(MXE_CONFIGURE_OPTS) \
-        AR='$(TARGET)-ar'
+        AR='$(TARGET)-ar' \
+        CFLAGS='-Wno-error=pointer-to-int-cast'
```

Comment:
This is not safe at all. Find the error in the code and fix it instead of ignoring the warning/error.